Tests/Resources/TemplateRenderingTest.php

```diff
@@ -239,6 +239,57 @@ public function testStepList_stepSkipped() {
 		$this->assertContains('<li class="craue_formflow_current_step">step2</li>', $renderedTemplate);
 	}
 
+	public function testRenderResetButton() {
+		$flow = $this->getFlowStub();
+
+		// first step
+		$flow->nextStep();
+
+		// By default contains, reset button and have two buttons
+		$renderedTemplate = $this->getTwig()->render(self::BUTTONS_TEMPLATE, array(
+			'flow' => $flow,
+		));
+		$this->assertContains('<button type="submit" class="craue_formflow_button_first" name="flow_renderingTest_transition" value="reset" formnovalidate="formnovalidate">start over</button>', $renderedTemplate);
+		$this->assertContains('<div class="craue_formflow_buttons craue_formflow_button_count_2">', $renderedTemplate);
+
+		// Force to true, contains reset button and have two buttons
+		$renderedTemplate = $this->getTwig()->render(self::BUTTONS_TEMPLATE, array(
+			'craue_formflow_render_reset_button' => true,
+			'flow' => $flow,
+		));
+		$this->assertContains('<button type="submit" class="craue_formflow_button_first" name="flow_renderingTest_transition" value="reset" formnovalidate="formnovalidate">start over</button>', $renderedTemplate);
+		$this->assertContains('<div class="craue_formflow_buttons craue_formflow_button_count_2">', $renderedTemplate);
```

Comment:
Lines 247-262 aren't needed since `testButtons` already covers rendering _with_ the reset button.